modules/tinymce/src/plugins/importcss/test/ts/browser/ImportCssGroupsPluginTest.ts

```diff
@@ -61,156 +37,156 @@ UnitTest.asynctest('browser.tinymce.plugins.importcss.ImportCssGroupsTest', (suc
       importcss_selector_converter: pluginSettings.importcss_selector_converter,
       importcss_exclusive: pluginSettings.importcss_exclusive,
       base_url: '/project/tinymce/js/tinymce'
-    }, () => onStepSuccess(), onStepFailure);
-  });
+    });
 
-  Pipeline.async({ }, [
+    Assertions.assertPresence(
+      `${assertion.choice.presence} should NOT be present`,
+      {
+        [assertion.choice.presence]: 0
+      },
+      TinyDom.body(editor)
+    );
+    TinyUiActions.clickOnToolbar(editor, 'button');
+    await pProcessNavigation(SugarDocument.getDocument(), assertion.navigation);
+    Arr.each(assertion.choice.keysBeforeExecute, (k) => TinyUiActions.keydown(editor, k));
+    TinyUiActions.keydown(editor, Keys.enter());
+    Assertions.assertPresence(
+      `${assertion.choice.presence} should now be present`,
+      {
+        [assertion.choice.presence]: 1
+      },
+      TinyDom.body(editor)
+    );
+    McEditor.remove(editor);
+  };
 
-    Log.step(
-      'TBA',
-      'importcss: content_css with three files, append false, groups with overall selector converter',
-      sTestEditorWithSettings(
-        {
-          navigation: [
-            { item: 'Other', subitems: [ 'h1.red.DDD', 'p.other.DDD', 'span.inline.DDD' ] },
-            { item: 'Advanced', subitems: [ 'h2.advanced.CCC', 'h3.advanced.CCC', 'h4.advanced.CCC' ] }
-          ],
-          choice: {
-            keysBeforeExecute: [ Keys.right() ],
-            presence: 'span.converted'
-          }
-        },
-        {
-          content_css: [
-            '/project/tinymce/src/plugins/importcss/test/css/basic.css',
-            '/project/tinymce/src/plugins/importcss/test/css/advanced.css',
-            '/project/tinymce/src/plugins/importcss/test/css/other-adv.css'
-          ],
-          importcss_append: false,
-          importcss_groups: [
-            { title: 'Advanced', filter: /.adv/, custom: '.CCC' },
-            { title: 'Other', custom: '.DDD' }
-          ],
+  it('TBA: content_css with three files, append false, groups with overall selector converter', () => pTestEditorWithSettings(
```

Comment:
Is it worth making `pTestEditorWithSettings` return an async thunk instead of returning the promise, so that we can unthunk it in all of the `it` calls?

Reply:
I didn't want to do that as it prevents additional tests/assertions being done here if required.